### PR TITLE
EW-1060 redirected the common cartridge export to the cc-microservice

### PR DESCRIPTION
**Pull request by Fshmit**
# Short Description

- The export endpoint is redirected to the Common Cartridge microservice. The export request now goes to the microservice instead of the schulcloud server.
- Information regarding the export of files was deleted, as the export of files is now implemented and executable. 

## Links to Ticket and related Pull-Requests
[Ticket](https://ticketsystem.dbildungscloud.de/browse/EW-1060)

- https://github.com/hpi-schul-cloud/schulcloud-server/pull/5351

## Checklist before merging

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] PO: Any deviation from requirements was agreed with Product-Owner / ticket author / support-team
- [ ] DEV: Every new component is implemented having accessibility in mind (e.g. aria-label, role property)

> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed

### CODE REVIEW

File: src/components/molecules/CommonCartridgeExportModal.vue [hunk 1]
```diff
@@ -41,9 +41,6 @@
 							{{ $t(`components.molecules.export.options.info`) }}
 							<br />
 							&middot;
-							{{ $t(`components.molecules.export.options.info.point1`) }}
```
Comment (MajedAlaitwniCap) [inline src/components/molecules/CommonCartridgeExportModal.vue:44]: should this be removed from language Files?

File: src/store/course-room-details.ts [hunk 2]
```diff
@@ -215,6 +216,22 @@ export default class CourseRoomDetailsModule extends VuexModule {
 					responseType: "blob",
 				}
 			);
+			// const response = await CoursesApiFactory(
```
Comment (MajedAlaitwniCap) [inline src/store/course-room-details.ts:219]: when should this be deleted?

**Review by MajedAlaitwniCap — CHANGES_REQUESTED** [review]

File: package.json [hunk 3]
```diff
@@ -13,7 +13,8 @@
 		"lint:fix": "npx eslint 'src/**/*.{ts,js,vue}' --fix",
 		"generate-client:server": "node generate-client.js -c openapitools-for-server.json",
 		"generate-client:filestorage": "node generate-client.js -u 'http://localhost:4444/api/v3/docs-json/' -p 'src/fileStorageApi/v3' -c 'openapitools-for-file-storage.json'",
-		"generate-client:h5p-editor": "node generate-client.js -u 'http://localhost:4448/api/v3/docs-json/' -p 'src/h5pEditorApi/v3' -c 'openapitools-for-h5p-editor.json'"
+		"generate-client:h5p-editor": "node generate-client.js -u 'http://localhost:4448/api/v3/docs-json/' -p 'src/h5pEditorApi/v3' -c 'openapitools-for-h5p-editor.json'",
+		"generate-client:common-cartridge": "node generate-client.js -u 'http://localhost:3350/api/v3/docs-json/' -p 'src/commonCartridgeApi/v3' -c 'openapitools-for-common-cartridge.json'"
```
Comment (psachmann) [inline package.json:17]: Deletion commend is missing and use the openapi-generator-cli directly in the same way we are generating the clients in the server.

File: openapitools-for-common-cartridge.json
Comment (psachmann) [inline openapitools-for-common-cartridge.json:1]: Create one openapitools.json file with the config.

**Review by psachmann — CHANGES_REQUESTED** [review]

**Review by psachmann — DISMISSED** [review]

File: src/store/course-room-details.ts (hunk 2 above)
Comment (psachmann) [reply src/store/course-room-details.ts]: Comment removed

**Review by psachmann — DISMISSED** [review]

**Review by MajedAlaitwniCap — DISMISSED** [review]

**Review by psachmann — APPROVED** [review]

**Review by MajedAlaitwniCap — APPROVED** [review]

**Pull request merged**